pkgs/top-level/all-packages.nix

```diff
@@ -13464,7 +13464,7 @@ with pkgs;
     libdrm = if stdenv.isLinux then libdrm else null;
     abiCompat = config.xorg.abiCompat # `config` because we have no `xorg.override`
       or (if stdenv.isDarwin then "1.18" else null); # 1.19 needs fixing on Darwin
-  } // { inherit xlibsWrapper; } );
+  } // { inherit xlibsWrapper; } ) [ "packages" ]);
```

Comment:
Need to remove the `"packages"` attribute from `xorg`'s scope. Otherwise it gets merged into the main attribute set and makes `lambdabot` fail. I don't yet know why it's not a problem to have `"callPackage"`, etc from `makeScope` 

https://github.com/NixOS/nixpkgs/blob/8014c7f186afb66ae746d6b1af9ce4510fb55eae/pkgs/top-level/splice.nix#L27-L28

https://github.com/NixOS/nixpkgs/blob/8014c7f186afb66ae746d6b1af9ce4510fb55eae/lib/customisation.nix#L194-L203

They should probably also be removed. Should this be done in `all-packages.nix` or `splice.nix`?